src/checkout-buttons/strategies/paypal-button-strategy.ts

```diff
@@ -87,7 +88,13 @@ export default class PaypalButtonStrategy extends CheckoutButtonStrategy {
             throw new NotInitializedError(NotInitializedErrorType.CheckoutButtonNotInitialized);
         }
 
-        return actions.request.post('/api/storefront/paypal-payment/', { merchantId })
+        return actions.request.post(`${this._host}/api/storefront/paypal-payment/`, {
+            merchantId,
+        }, {
+            headers: {
+                'X-API-INTERNAL': 'This API endpoint is for internal use only and may change in the future',
```

Comment:
Oh I forgot, you can actually import this constant `INTERNAL_USE_ONLY` from `'common/http-request'`

Reply:
Oh, TIL. Will do.